lua/pl/path.lua

```diff
@@ -49,7 +49,7 @@ path.currentdir = currentdir
 path.chdir = lfs.chdir
 
 
---- is this a directory?
+--- Is this a directory ?
```

Comment:
the inserted space is not correct here. Nor in the other places, can you please remove them?